### PR TITLE
[WIP] UI made, now will start with backend integration

### PR DESCRIPTION
**Pull request by prateekinstillion**
#### What does this PR do?
(A brief explanation synthesizing the feature, bug or fix.)
- UI components laid for chat page

## Checklist

(Go over all the following points, and put an `x` in all the boxes that apply.)
(If you're unsure about any of these, don't hesitate to ask. We're here to help!)

- [x ] My code follows the code style of this project.
- [ ] Make sure that you don't have **[WIP]** if you have completed your PR.
- [x] **I am choosing the correct base branch to merge**. 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] And more importantly, I've had fun! :beer:

#### Where should the reviewer start?
(Explain where the reviewer should start in order to review the whole addition or subtractions.)
:checkered_flag:

#### How should this be manually tested?
(List of steps to reproduce, corroborate or tests to run. Write this section clear enough so that external users can also follow it and test the fix.)
1. :one:
2. :two:
3. :tada:

#### Any background context you want to provide?
(Any information regarding the PR that the reviewer should know.)
:construction:

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #x

#### Screenshots
Screenshot from 2019-07-10 02-21-23

:camera:

#### Questions
(If available, to another developer or reviewer.)
:question:

### CODE REVIEW

**Comment by abhi18av** [comment]
@prateekinstillion , please refer this 

https://github.com/pesto-students/batch-8-NuSlack/pull/25#issuecomment-510283312

File: client/src/App.js [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import "./App.css";
```
Comment (abhi18av) [inline client/src/App.js:3]: @prateekinstillion , it seems that @rollaball  is using `SCSS` for the same, so either remove the `global CSS` entirely or atleast re-factor to reach a common standard.

File: client/src/dummy.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (abhi18av) [inline client/src/dummy.json:1]: @prateekinstillion , try and avoid commiting `dummy` files to the main repo, and focus on the tests to make sure things are working as expected.

File: client/src/utils/isEmpty.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const isEmpty = value =>
```
Comment (abhi18av) [inline client/src/utils/isEmpty.js:1]: @prateekinstillion , bro this is doing too many things for a good arrow function. Please try to improvise towards more readability here.

**Review by abhi18av — COMMENTED** [review]
I've made some suggestions regarding the readability.

**Comment by abhi18av** [comment]
@prateekinstillion , please also try to stay consistent with the `PR` guide at the `ReadMe` of this repo.

File: client/src/utils/isEmpty.js (hunk 3 above)
Comment (prateekinstillion) [reply client/src/utils/isEmpty.js:1]: It is generic bro. It can be used as and when needed

File: client/src/dummy.json (hunk 2 above)
Comment (prateekinstillion) [reply client/src/dummy.json:1]: OK. noted

**Pull request closed without merging**